yellowbrick/gridsearch/pcolor.py

```diff
@@ -0,0 +1,225 @@
+"""
+Colorplot visualizer for gridsearch results.
+"""
+
+import time
+import numpy as np
+
+from .base import GridSearchVisualizer
+from ..exceptions import YellowbrickValueError
+
+from sklearn.metrics.pairwise import pairwise_distances
+from sklearn.preprocessing import LabelEncoder
+from matplotlib.mlab import griddata
+
+
+## Packages for export
+__all__ = [
+    "GridSearchColorPlot",
+    "gridsearch_color_plot"
+]
+
+
+##########################################################################
+## Dimension reduction utility
+##########################################################################
+
+def param_projection(cv_results, param_1, param_2):
+    """
+    Projects the grid search results onto 2 dimensions.
+
+    The display value is taken as the max over the non-displayed dimensions.
+
+    Parameters
+    ----------
+    cv_results : dict
+        A dictionary of results from the `GridSearchCV` object's `cv_results_`
+        attribute
+
+    param_1 : string
+        The name of the parameter to be visualized on the horizontal axis.
+
+    param_2 : string
```

Comment:
I can see the motivation for `param_1` vs `param_2` - but because these names are connected to the vertical and horizontal axes, maybe it would be easier to name them "x_param" and "y_param" - I know this is not really the x and y axes but they do it on bar graphs so I think it would be ok here.